src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -153,4 +158,44 @@ private static boolean containerVersion(String plugin) {
         return false;
     }
 
+    public static String getGradleSettingsCmd(Project project) {
+        GradleProjectSettings gradleProjectSettings = GradleSettings.getInstance(project).getLinkedProjectSettings(project.getBasePath());
+
+        if (gradleProjectSettings.getDistributionType().isWrapped()) {
+            // a wrapper will be used
+            String wrapperPath = getLocalGradleWrapperPath(project);
+            if (wrapperPath != null) {
+                return wrapperPath;
+            }
+        }
+        else if (DistributionType.LOCAL.equals(gradleProjectSettings.getDistributionType())) {
+            // local gradle to be used
+            String gradleHome = gradleProjectSettings.getGradleHome(); //it is null when the path to gradle is invalid
+            if (gradleHome != null) {
+                String gradlePath = getCustomGradlePath(gradleHome);
+                if (gradlePath != null) {
+                    return gradlePath;
+                }
+            }
+        }
+        return "gradle"; // default gradle
+    }
+
+    private static String getLocalGradleWrapperPath(Project project) {
+        String gradlew = SystemInfo.isWindows ? ".\\gradlew.bat" : "./gradlew";
+        File file = new File(project.getBasePath() + File.separator + gradlew);
```

Comment:
Suggestion, we usually use the File(File, String) constructor in a case like this.